### DISCUSSION TITLE
Enable adding guests by email to Google Calendar event

### DISCUSSION THREAD

**Discussion by Aleksander-Drozd** [add2cal/add-to-calendar-button#465] [Ideas / Requests]
Hi, I've already read [attendee param](https://add-to-calendar-button.com/configuration) docs and if I understand correctly it doesn't allow adding any user by an email. It is possible that you add support for `add` query param that allows doing that?

`add` param docs - https://github.com/InteractionDesignFoundation/add-event-to-calendar-docs/blob/main/services/google.md
demo: https://calendar.google.com/calendar/render?action=TEMPLATE&text=Event+title&add=example@example.com

**Comment by jekuer** [comment]
Hi @Aleksander-Drozd ,

Thank you for your comment.
May I ask about your use case?
Usually, defining other participants for the event does not make sense in an Add-to-Calendar-Case. This is rather something the organizer of an event defines at a manual event - or the user does manually in his calendar app.

**Reply by Aleksander-Drozd** [reply]
It's very specific to client of a company I work for. Managers create calendar events based on internal meetings for employees working in the field. The goal is for the employee to see a meeting created by the manager in their calendar. They can do this manually but it's faster when guest is added automatically especially if they create 20-30 of such meetings a day.

**Reply by jekuer** [reply]
Thanks!

How I see it:
- still no case which makes sense for this Add-to-Calendar tool. We can add it anyway, since the current attendee input is also only some bad workaround - but will have very low priority.
- for your case, I would not recommend using this package. The Add to Calendar Button is useful in cases, where you have no idea about the user's setup in terms of which calendar he/she uses, on which device, OS, ... - in your case, you know all of this. Here, it would be way more stable and cleaner if you simply gather the data from the internal meeting and push it directly into one big Google Calendar (or Outlook - whatever your client uses) via Google's API (or Microsoft respectively). This big calendar can be the manager's one - but still - going via the API would be more stable and also provides you with more functionality.
- Even if you do not want to use the API, you could still directly generate the Add-to-Calendar-link directly. You do not need all the magic of this package, because, again, you know who is operating the app and can tailor it directly to his case. 😃 
- But maybe I misunderstood.

**Reply by Aleksander-Drozd** [reply]
Generating calendar links directly turned out to be a good idea, [calendar-link](https://www.npmjs.com/package/calendar-link) supports `add` query parameter. Thanks for help.
